dashboard/src/components/localhost/AttendeeRequestList.vue

```diff
@@ -225,35 +221,30 @@ const props = defineProps({
   },
 })
 
-const emit = defineEmits(['updateRequest'])
```

Comment:
Why remove this? this was a crucial fix to the bug which was not updating the status of requests in Localhost Dashboard page